src/components/productdisplayitem.main.jsx

```diff
@@ -63,6 +64,8 @@ const zoomArray = [
   'code',
 ];
 
+const configurationPrefix = epConfig.cartItemModifier.prefix;
```

Comment:
Not necessary as you can reuse the existing config and pull what you need later in your block with:
`Config.cartItemModifier.prefix`

Reply:
Removed